corehq/apps/api/odata/utils.py

```diff
@@ -23,3 +24,24 @@ def get_case_type_to_properties(domain):
             cleaned_case_property = export_group_schema.label.replace('_', '')
             case_type_to_properties[case_type].append(cleaned_case_property)
     return dict(case_type_to_properties)
+
+
+def get_xmlns_by_app(domain, app_id):
+    app = get_current_app(domain, app_id)
+    return [form.xmlns.split('/')[-1] for form in app.get_forms()]
+
+
+def get_properties_by_xmlns(domain, app_id, xmlns):
+    complete_xmlns = 'http://openrosa.org/formdesigner/' + xmlns
```

Comment:
If a user uploads their own XML this pattern isn't guaranteed to be correct (they can set xmlns to whatever they want)